testgrid/cmd/configurator/prow.go

```diff
@@ -139,8 +150,18 @@ func applySingleProwjobAnnotations(c *configpb.Configuration, pc *prowConfig.Con
 	if tn, ok := j.Annotations[testgridTabNameAnnotation]; ok {
 		tabName = tn
 	}
+
+	description := fmt.Sprintf("prowjob_name: %v", j.Name)
+	if pac.prowJobURLPrefix != "" {
+		url := pac.prowJobURLPrefix + strings.TrimPrefix(j.SourcePath, pac.prowJobConfigPath)
+		description = fmt.Sprintf("prowjob_name: %v, prowjob_config_url: %v", j.Name, url)
+	}
 	if d := j.Annotations[descriptionAnnotation]; d != "" {
-		description = d
+		if pac.updateDescription {
+			description = fmt.Sprintf("%v, prowjob_description: %v", description, d)
```

Comment:
If you want a newline btw, you should be able to add a `\n`!

Reply:
thanks for the tip, I'll change this